docs/lessons/toolkit-tooling/lesson-424-a-convergence-step-scoped-to-the-creation-diff-repairs-nothing.md

```
@@ -0,0 +1,80 @@
+status: active
+created: "2026-09-04"
+owner: manu
+category: toolkit-tooling
+tags: [kubelab, toolkit-tooling, reconciler, idempotence, gitea, guards]
+---
+
+# A convergence step scoped to the creation diff never repairs what already exists
+
+**Context**: The Gitea reconciler's `reconcilers` team granted `repo.code -> write`
+over zero repositories, because `includes_all_repositories` defaults to `false`.
+#1611 fixed it: `create_team` sends the flag, `edit_team` widens a team that
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)